apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -350,7 +345,235 @@ paths:
         '200':
           description: Success
           content:
-            application/fhir+json:
-              schema: { $ref: '#/components/schemas/FHIR-JSON-RESOURCE' }
-            application/fhir+xml:
-              schema: { $ref: '#/components/schemas/FHIR-XML-RESOURCE' }
+            application/x-www-form-urlencoded:
+              schema:
+  /v2/fhir/Claim/_search:
+      post:
+        tags: [v2,Claim]
+        summary: 'search-type: Search for Claim instances'
+        description: This is a search type
+        requestBody:
+          content:
+            application/x-www-form-urlencoded:
+              schema:
+                type: object
+                properties:
+                  mbi:
+                    type: string
+                    description: |-
+                      **NOTE: THIS IS A REQUIRED FIELD**
```

Comment:
If this is required, it should have that in the "required" syntax rather than a bolded note.